tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,11 @@ def test_removing_not_found_returns_none():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == None
```

Comment:
Nice assert, we definitely want to confirm our return value is `None`. We generally also want to confirm that there weren't side affects or changes to the input that we didn't intend. What could we check about the vendor's inventory to confirm that its content didn’t change?

Reply:
That makes sense. Then we would check the following:
assert "a" in vendor.inventory
assert "b" in vendor.inventory
assert "c" in vendor.inventory
assert len(vendor.inventory) == 3

I have an additional question, isn't it better to write "assert result is None"? Or I am thinking about "assert not result". Which of the three ways is best?

Reply:
If we want to know if a variable is pointing to specifically `None`, we want to use `assert result is None`. Any comparison directly to `None` we want to use the `is`/`is not` operators with. `assert not result` is less specific, since the assert will pass if the variable points to a falsy value like `0` or an empty collection as well as `None`.